docs/sources/administration/image_rendering.md

```diff
@@ -194,3 +194,21 @@ it's available for the Grafana process, e.g.
 ```bash
 export GF_RENDERER_PLUGIN_CHROME_BIN="/usr/bin/chromium-browser"
 ```
+
+### Grafana image renderer plugin and certificate signed by internal CAs
+
+If you are using the Grafana image renderer with a Grafana server that uses a certificate signed by a custom CA (e.g. a company-internal CA) that is not known to the nss trust store shipped with headless chrome, rendering images will fail and you will see messages like this in the Grafana log:
+
+```t=2019-12-04T12:39:22+0000 lvl=eror msg="Render request failed" logger=rendering error=map[] url="https://192.168.106.101:3443/d-solo/zxDJxNaZk/graphite-metrics?orgId=1&refresh=1m&from=1575438321300&to=1575459921300&var-Host=master1&panelId=4&width=1000&height=500&tz=Europe%2FBerlin&render=1" timestamp=0001-01-01T00:00:00.000Z
+t=2019-12-04T12:39:22+0000 lvl=eror msg="Rendering failed." logger=context userId=1 orgId=1 uname=admin error="Rendering failed: Error: net::ERR_CERT_AUTHORITY_INVALID at https://192.168.106.101:3443/d-solo/zxDJxNaZk/graphite-metrics?orgId=1&refresh=1m&from=1575438321300&to=1575459921300&var-Host=master1&panelId=4&width=1000&height=500&tz=Europe%2FBerlin&render=1"
+t=2019-12-04T12:39:22+0000 lvl=eror msg="Request Completed" logger=context userId=1 orgId=1 uname=admin method=GET path=/render/d-solo/zxDJxNaZk/graphite-metrics status=500 remote_addr=192.168.106.101 time_ms=310 size=1722 referer="https://grafana.xxx-xxx/d/zxDJxNaZk/graphite-metrics?orgId=1&refresh=1m"
+```
+
+In this case you will have to add the certificate to the trust store. If you have the certificate file for the internal root CA in the file `internal-root-ca.crt.pem`, use these commands to create a user specific NSS trust store for the Grafana user (`grafana` for the purpose of this example) and execute the following steps:
+
+```[root@server ~]# [ -d /usr/share/grafana/.pki/nssdb ] || mkdir -p /usr/share/grafana/.pki/nssdb
+[root@merver ~]# certutil -d sql:/usr/share/grafana/.pki/nssdb -A -n internal-root-ca -t C -i /etc/pki/tls/certs/internal-root-ca.crt.pem
+[root@server ~]# chown -R grafana: /usr/share/grafana/.pki/nssdb
+```
+
+The same procedure should be usable for self-signed certificates. 
```

Comment:
```suggestion
Use the same procedure for self-signed certificates. 
```

Reply:
Why not put this statement in the paragraph at the beginning of this section? Does this mean that the procedure can be used for all certificates? If so, we should say that. It is confusing to split the scope of this fix into two parts, one at the beginning and one at the end.

Reply:
@oddlittlebird: Thanks for your suggestions, I just pushed an update. 

Regarding your last remark I came to the conclusion that I better leave that part out entirely:

- I have not tested it with self-signed certificates, so I cannot tell for sure if it works
- It may or may not be a bad idea to set the 'T' bit in the NSS trust store for a self-signed certificate (I'm not that deep into NSS that I can fully judge the consequences - it **should** not do any harm but I'd rather err on the safe side
- Hardly anyone uses self-signed certificates (call me an optimist)